src/ServicePulse.Host/vue/src/components/failedmessages/OrderBy.vue

```diff
@@ -31,14 +39,18 @@ function loadSavedSortOption() {
     const sortBy = getSortOptions().find((sort) => {
       return sort.description.toLowerCase() === criteria.toLowerCase();
     });
-    return { sort: getSortFunction(sortBy.selector, direction), dir: direction, description: sortBy.description };
+    if (sortBy) {
+      sortBy.sort = getSortFunction(sortBy.selector, direction);
+      sortBy.dir = direction;
+      return sortBy;
+    }
```

Comment:
this is modifying the original referenced object, whereas the old code was not. I would prefer to have
```suggestion
    if (sortBy) {
      return { ...sortBy, sort: getSortFunction(sortBy.selector, direction), dir: direction }
    }
```

Reply:
Done